pandas/io/common.py

```diff
@@ -161,6 +161,7 @@ def is_fsspec_url(url: FilePathOrBuffer) -> bool:
     return (
         isinstance(url, str)
         and "://" in url
+        and not " " in url
```

Comment:
@martindurant

Reply:
A space is allowed in URLs for at least some storage systems

Reply:
Thanks, any other ideas how to fix #36271 ? All my json strings with URLs in them receive errors and I downgraded to pandas==1.0.5

Reply:
So, a space isn't necessary for valid JSON either. Perhaps the only way to tell is to try to parse the string?

Reply:
I updated the PR to reflect what you say, I'm now trying to parse the string as json and if it succeeds, it's not a URL. How about that?

Reply:
Sounds reasonable to me - I don't expect any URL is also valid JSON. You could conceivable make a regex to check for JSON-ness (`"^\s*[\[{]"` maybe?), but probably calling JSON decode is faster and more correct.

(I simply didn't know that you can pass a string directly to read_json, as opposed to a StringIO file-like)